instrumentation/opentelemetry-instrumentation-aiokafka/src/opentelemetry/instrumentation/aiokafka/utils.py

```diff
@@ -0,0 +1,275 @@
+import json
+from logging import getLogger
+from typing import Any, Awaitable, Callable, Dict, List, Optional, Tuple, Union
+
+import aiokafka
+from aiokafka import ConsumerRecord
+
+from opentelemetry import context, propagate, trace
+from opentelemetry.context import Context
+from opentelemetry.propagators import textmap
+from opentelemetry.semconv.trace import SpanAttributes
+from opentelemetry.trace import Tracer
+from opentelemetry.trace.span import Span
+
+_LOG = getLogger(__name__)
+
+
+class AIOKafkaPropertiesExtractor:
```

Comment:
I think this class is not needed and you can just have all the methods as free functions (prefixed with `_`).

Reply:
done